src/utils/project.ts

```diff
@@ -0,0 +1,10 @@
+import Project from "../resources/Project";
+
+export function findProjectById(projects: Project[], id: number): Project {
+  const proj = projects.filter(function (project) {
+    // eslint-disable-next-line
```

Comment:
If you disable eslint, you must include the reason as well.
```suggestion
```

Reply:
i'm disabling it to avoid a warning from eqeqeq about using == when it should be ===, but using === causes the app to crash even when casting project.id and id as numbers (though they are already typed as such). I spent some time when initially building the ReservationForm but could not determine why === did not work.